resources/type-system/subtyping.md

```diff
@@ -43,6 +44,8 @@ The set of types under consideration are as follows:
 - Function types
   - `U Function<X0 extends B0, ...., Xk extends Bk>(T0 x0, ...., Tn xn, [Tn+1 xn+1, ..., Tm xm])`
   - `U Function<X0 extends B0, ...., Xk extends Bk>(T0 x0, ...., Tn xn, {Tn+1 xn+1, ..., Tm xm})`
+- Record types
+  - `(T0 x0, ...., Tn xn, {Tn+1 xn+1, ..., Tm xm})`
```

Comment:
Can drop the `x0`..`xn` names here. Or maybe say that they are optional and have no effect on anything.